regimes/au/invoices.go

```diff
@@ -0,0 +1,97 @@
+package au
+
+import (
+	"github.com/invopop/gobl/bill"
+	"github.com/invopop/gobl/l10n"
+	"github.com/invopop/gobl/num"
+	"github.com/invopop/gobl/org"
+	"github.com/invopop/gobl/tax"
+	"github.com/invopop/validation"
+)
+
+type invoiceValidator struct {
```

Comment:
This is fine, but we're moving away from using validator structs to just have method calls. There isn't much value in the struct in the end.